shells/chrome/README.md

```diff
@@ -2,11 +2,11 @@
 
 The source code for this extension has moved to `shells/webextension`.
 
-Modify the source code there and then rebuild this extension by running `node build.js` in this directory.
+Modify the source code there and then rebuild this extension by running `yarn run build:extension:chrome`.
 
 ## Testing in Chrome
 
 You can test a local build of the web extension like so:
 
- 1. Build the extension: `node build`
- 1. Follow the on-screen instructions.
\ No newline at end of file
+ 1. Build the extension: `yarn run build:extension:chrome`
```

Comment:
nit: `node build` and `yarn run build:extension:chrome` are the same thing, except the latter needs to be run from the root directory. I don't mind changing the README but we should be clear I guess that the yarn command should not be run from the current dir.